alerting/src/test/kotlin/org/opensearch/alerting/action/GetFindingsRequestTests.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.alerting.action
```

Comment:
NIT: This test class is present in common utils along with the request class. We shouldn't have this here as these tests would break if anything is changed in common utils

Reply:
Oh this test was present in Alerting main [here](https://github.com/opensearch-project/alerting/blob/main/alerting/src/test/kotlin/org/opensearch/alerting/action/GetFindingsRequestTests.kt)